bin/jobs_functions.js

```diff
@@ -77,7 +78,7 @@ module.exports = {
                 break;
             default:
                 util.error("Unknown operatation");
```

Comment:
Maybe you could fix this spelling error while you're here

Reply:
Oh yes. Sharp eye :)